ui/src/datasets.tsx

```diff
@@ -225,9 +236,41 @@ export function Datasets() {
           </Paper>
         </Grid>
         <Grid item xs={1}>
-          <Stack direction="row" alignItems="baseline">
-            <Typography variant="h4">3. Values</Typography>
-            <Typography variant="h5">(Columns)</Typography>
+          <Stack
+            direction="row"
+            alignItems="center"
+            justifyContent="space-between"
+          >
+            <Stack direction="row" alignItems="center">
+              <Typography variant="h4" mr={1}>
```

Comment:
Is the margin here still necessary now that the checkbox is right aligned?

Reply:
I think this was an issue with the previous version too, but without the margin, there isn't a gap between "Values" and "(Columns)". Even if you add trailing and leading spaces, they appear as a connected word. I just added the margin for a stylistic reason, but can remove it if you feel it's unnecessary.

Reply:
Ah, I see. That's fine. I had misread anyways and thought this was after "(Columns)".